### DISCUSSION TITLE
when your default color theme is sweetkind then youe cant switch between appearence modes light and dark

### DISCUSSION THREAD

**Discussion by plugeit** [TomSchimansky/CustomTkinter#335] [Q&A] [answered]
ctk.set_appearance_mode("Dark")
ctk.set_default_color_theme("sweetkind")

but if i want to change it 
ex: 
ctk.set_appearence_moce('Light')

!!!you can't do that!!!

**Comment by plugeit** [comment]
you need to destroy the tkinter window and then you can do that
[scr.destroy(),do_this()]

**Comment by felipetesc** [comment, marked as answer]
Hello. sweetkind only has a dark theme. I'm working to create the white (hateful for my eyes) version

**Comment by felipetesc** [comment]
![sweetkind_white](https://user-images.githubusercontent.com/2336812/182437240-ac3efd9d-2442-4903-a431-a871e7061dce.png)

**Comment by felipetesc** [comment]
Can you close this one if the question is answered ? Thanks.
